test/parallel/test-child-process-fork-regr-gh-2847.js

```diff
@@ -35,9 +40,14 @@ const server = net.createServer(function(s) {
 
     // Errors can happen if this connection
     // is still happening while the server has been closed.
+    // https://github.com/nodejs/node/issues/3635#issuecomment-157714683
+    // If a previous request caused the worker __and__ the server to close
```

Comment:
`- If a previous request...`

Basically can it be really clear which explanation is for which errors?